src/clients.ts

```diff
@@ -44,8 +44,12 @@ function log(text: string = '') {
 }
 
 function getRepositoryName(): string {
-  const output = spawnSync('gh', ['repo', 'view', '--json', 'nameWithOwner'], { stdio: ['ignore', 'pipe', 'inherit'] }).stdout.toString('utf-8');
-
+  const spawn = spawnSync('gh', ['repo', 'view', '--json', 'nameWithOwner'], { stdio: ['ignore', 'pipe', 'pipe'] });
+  const stderr = spawn.stderr?.toString();
+  if (stderr) {
```

Comment:
Not a fan of this. Can’t we get an exit code?

Reply:
Sure. We can throw on non 0 exit codes. Are you cool with keeping the stderr output in the message or would you rather not `'pipe'` it?

```ts
const spawn = spawnSync('gh', args, { input: value, stdio: ['pipe', 'inherit', 'inherit'] });
  if (spawn.status !== 0) {
    throw new Error(`Failed to store secret '${name}' in repository '${repository}'. ${spawn.stderr?.toString()}`);
  }
```